src/features/identityCheck/pages/confirmation/__test__/IdentityCheckHonor.test.tsx

```diff
@@ -75,4 +78,21 @@ describe('<IdentityCheckHonor/>', () => {
       expect(navigate).toHaveBeenCalledWith('BeneficiaryAccountCreated')
     })
   })
+
+  it('should navigate to BeneficiaryAccountCreated on postHonorStatement request success if user s credit is expired (non beneficiary)', async () => {
```

Comment:
On veut tester aussi qu'on navigue vers `BeneficiaryAccountCreated` en cas de crédit expiré ? Comme le test du dessus ?

Reply:
Ce test permet plutôt de vérifier que l'on ne navigue pas vers BeneficiaryAccountCreated en cas de crédit expiré.
Tu le trouve redondant avec un autre test ?

Reply:
Non c'est plutôt qu'il y a une coquille dans le nom du test du coup "should navigate to BeneficiaryAccountCreated" ça devrait plutôt être "should NOT navigate to...".
Et dans ce cas là ce serait pas mal de soit nommer le test "should navigate to next screen..." puisque c'est ce qui est testé précisément, ou alors ajouter un `expect(navigate).not.toHaveBeenCalledWith('BeneficiaryAccountCreated')`.
T'en penses quoi ?

Reply:
Ah oui je comprends mieux, effectivement le titre n'est pas cohérent, je change ça !